topcoffea/data/TauSF/README.md

```diff
@@ -0,0 +1,8 @@
+# JSONs containing corrections and systematics
+
+Apart from the files starting with `TauFakeSF_*`, the documentation and the source of the files contained here is listed TAU POG GH repo: https://github.com/cms-tau-pog/TauIDSFs.git. When necessary, the central files where translated in the legacy JSON format used for TOP-22-006.
```

Comment:
What is meant by "legacy JSON format"? And I'm curious what's involved in the conversion? Also, I think "where" is supposed to read "were" in that last sentence?

Reply:
@anpicci @kmohrman did we want to edit the text here to remove the word "legacy"?

Reply:
@anpicci I think this is the last item in this PR. Do you have a chance to address it?

Reply:
@bryates if you are talking about the scripts to translate the TAU POG files in the jsons used in the framework, @jlawrenc should have them at his disposal

Reply:
I think it was just @kmohrman's concern of using the word `legacy`, since that implies pre-UL.

Reply:
@kmohrman would TOP-22-006 work in place of legacy?